@here/harp-mapview/lib/poi/PoiRendererFactory.ts

```diff
@@ -21,7 +18,7 @@ export class PoiRendererFactory {
      * @param textCanvas -
      * @returns
      */
-    createPoiRenderer(textCanvas: TextCanvas): PoiRenderer {
-        return new PoiRenderer(this.m_mapView, textCanvas);
+    createPoiRenderer(): PoiRenderer {
+        return new PoiRenderer(this.m_mapView);
```

Comment:
update documentation

Reply:
I removed the whole PoiRendererFactory, as now without the dependency on textCanvas we only need one instance per mapview, and I think it might solve the dissappearing Marker issue I am debugging